src/views/partials/dashboard/file_as_an_authorised_agent.njk

```diff
@@ -1,11 +1,22 @@
 <div class="govuk-summary-card">
     <div class="govuk-summary-card__title-wrapper">
-        <h2 class="govuk-summary-card__title govuk-!-margin-top-0">
-            {{lang.file_as_an_authorised_agent}}
-            <strong class="govuk-tag govuk-tag--blue govuk-!-margin-top-2">
-                {{lang.coming_soon}}
-            </strong>
-        </h2>
+
+        {% if agentStatus === AcspStatus.SUSPENDED %}
+            <h2 class="govuk-summary-card__title govuk-!-margin-top-0">
+                {{lang.file_as_an_authorised_agent}}
+                <strong class="govuk-tag govuk-tag--red govuk-!-margin-top-2">
+                    {{lang.cant_file_suspension}}
+                </strong>
+            </h2>
+        {% else %}
+            <h2 class="govuk-summary-card__title govuk-!-margin-top-0">
+                {{lang.file_as_an_authorised_agent}}
+                <strong class="govuk-tag govuk-tag--blue govuk-!-margin-top-2">
+                    {{lang.coming_soon}}
+                </strong>
+            </h2>
+        {% endif %}
```

Comment:
Could maybe nest just the strong element? So we don't duplicate the `h2` and `file_as_an_authorised_agent` text?

```suggestion
        <h2 class="govuk-summary-card__title govuk-!-margin-top-0">
            {{lang.file_as_an_authorised_agent}}
            {% if agentStatus === AcspStatus.SUSPENDED %}
                <strong class="govuk-tag govuk-tag--red govuk-!-margin-top-2">
                    {{lang.cant_file_suspension}}
                </strong>
            {% else %}
                <strong class="govuk-tag govuk-tag--blue govuk-!-margin-top-2">
                    {{lang.coming_soon}}
                </strong>
            {% endif %}
        </h2>
```

Reply:
Thanks, updated. The h2 is no longer duplicated. Might be better to keep the {{lang.file_as_an_authorised_agent}} duplicated as is should become a link in the future but not when the ACSP is suspended.